RxExample/RxExample/Examples/GeolocationExample/GeolocationViewController.swift

```diff
@@ -14,23 +14,21 @@ import CoreLocation
 #endif
 
 private extension Reactive where Base: UILabel {
-    var driveCoordinates: UIBindingObserver<Base, CLLocationCoordinate2D> {
+    var coordinates: UIBindingObserver<Base, CLLocationCoordinate2D> {
         return UIBindingObserver(UIElement: base) { label, location in
             label.text = "Lat: \(location.latitude)\nLon: \(location.longitude)"
         }
     }
 }
 
 private extension Reactive where Base: UIView {
-    var driveAuthorization: UIBindingObserver<Base, Bool> {
-        return UIBindingObserver(UIElement: base) { view, authorized in
-            if authorized {
-                view.isHidden = true
-                view.superview?.sendSubview(toBack:view)
+    func subviewPresence(_ subview: UIView) -> UIBindingObserver<Base, Bool> {
+        return UIBindingObserver(UIElement: base) { view, show in
+            if !show {
+                subview.removeFromSuperview()
             }
             else {
-                view.isHidden = false
-                view.superview?.bringSubview(toFront:view)
```

Comment:
Hi @Czajnikowski ,

are you suggesting to do that vs `rx.hidden` as an optimization? 

I would ideally want to reduce the amount of code as much as we can. You PR clearly improves the situation from where it was, but since we have started on this road, I would want to improve this part as much as we can.